composer.json

```diff
@@ -35,6 +35,7 @@
         "doctrine/phpcr-odm": "^1.3|^2.0",
         "jackalope/jackalope-doctrine-dbal": "^1.1.5",
         "ocramius/proxy-manager": "^1.0|^2.0",
+        "phpstan/phpdoc-parser": "^0.4",
```

Comment:
i think that this should stay in the `require` section, not `require-dev`

Reply:
You are totally right. No idea how it ended up in the dev dependencies.